addons/website/controllers/backend.py

```diff
@@ -73,3 +73,23 @@ def website_track_installing_modules(self, selected_features, total_features=Non
             'nbInstalled': total_features - len(features_not_installed)
         }
         return features_info
+
+    @http.route('/website/set_default_social_image', type='jsonrpc', auth='user')
+    def set_default_social_image(self, website_id, img_url):
+        website = request.env['website'].browse(website_id)
+        if not website or not website.exists():
+            return {'error': 'Website not found'}
+        website.social_default_image_src = img_url
+        return True
+
+    @http.route(
+            '/website/unset_has_social_default_image',
+            type='jsonrpc',
+            auth='user',
+        )
+    def unset_has_social_default_image(self, website_id):
+        website = request.env['website'].browse(website_id)
```

Comment:
Is there a reason we do not want to rely on `env['website'].get_current_website()` ?

Reply:
No, and @mash-odoo fixed it, but in the end, we extract all default_social_image-related changes into another task: https://github.com/odoo/odoo/pull/212662/commits/34e42e3e40582ffb01cae9c82a60c346361e3a86